### ISSUE TITLE
2 problems: can't arm /disarm - no preview

### ISSUE THREAD

**Issue by unecc** [twrecked/hass-aarlo#818]
Hi, 
i've a problem with aarlo.
Arm/disarm doesn't works

![image](https://github.com/twrecked/hass-aarlo/assets/134554559/6e9626b4-a615-4e1a-9b90-03b7e6f38169)

when i click on little icon of arlo camera

![image](https://github.com/twrecked/hass-aarlo/assets/134554559/5ddca88b-04fe-4b08-ae59-465708d2762a)

preview doesn't works

![image](https://github.com/twrecked/hass-aarlo/assets/134554559/a2b7d9d0-3771-4651-a4a1-bcd7ae953a2e)

How can i solve it? Thanks!

**Comment by Avatar1976** [comment]
Can also confirm why I came here that location control panels have vanished.
My config:
![Screenshot_20231017_070858_Home Assistant](https://github.com/twrecked/hass-aarlo/assets/47725430/f6343acd-c739-4430-a042-d2dd271522f6)

**Comment by twrecked** [comment]
Can you [turn on](https://github.com/twrecked/hass-aarlo#basic-debug) and [collect](https://github.com/twrecked/hass-aarlo#bug-reports) some debug.

If you can do it around the time you press the alarm button and the camera button that would be great.

**Comment by Avatar1976** [comment]
Not really, the whole location control panel entity has vanished, I have pro5(S) cameras that don't use a base station their armed status is based off location.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by unecc** [comment]
Why u use my post???

**Comment by Avatar1976** [comment]
> Why u use my post???

Because I have one of your two problems? Did you want me to log a faulty separately?

**Comment by unecc** [comment]
Your problem is different. Use this. I will open an other post

**Comment by platini76** [comment]
I have the same preview problem
